packages/emotion/native/Css.ml

```diff
@@ -238,6 +238,7 @@ let pp_rules hash rules =
   Printf.sprintf "%s %s" declarations selectors
 
 type declarations =
+  | Global of rule list
```

Comment:
@davesnx I know that you approved it, but I have a request change to myself 😆 

Maybe Globals is a more precisely name for it:
```suggestion
  | Globals of rule list
```

Reply:
Sounds good

Reply:
@davesnx Done ✅